src/desktops/dskSelectMap.cpp

```diff
@@ -91,6 +96,10 @@ dskSelectMap::dskSelectMap(const CreateServerInfo& csi)
     AddTextButton(4, 590, 530, 200, 22, TC_GREEN2, _("Load game..."), NormalFont);
     // "Weiter"
     AddTextButton(5, 590, 560, 200, 22, TC_GREEN2, _("Continue"), NormalFont);
+    // random map generation
+    AddTextButton(6, 380, 530, 150, 22, TC_GREEN2, _("Create Random"), NormalFont);
+    // random map settings
+    AddTextButton(7, 540, 530, 40, 22, TC_GREEN2, _("..."), NormalFont);
```

Comment:
Ah ok, now I get it. I'll check how the things with the add-ons work as soon as I get some more time to spend here. Since there're quite a bunch of settings, I'd probably have to include another category for random map generation.